components/http_client.rst

```diff
@@ -116,6 +116,14 @@ immediately instead of waiting to receive the response::
 This component also supports :ref:`streaming responses <http-client-streaming-responses>`
 for full asynchronous applications.
 
+.. caution::
```

Comment:
this is too early and too scary in the page :)
I would try to make the description more... description.
We feel the stress when reading it, but there is nothing special here, this is normal operations. It needs to be better described of course.

Reply:
Agree for the scary thing :D

Where do you think it would fit the best in the doc?
IMHO it should be mentioned ASAP.

Reply:
What about adding a Troubleshooting headline + a subsection and mentioning not just the solution but also the WHY you came to this point?